### PR TITLE
Removed dependence on DataModel Services

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by Benestar** [comment]
-2 this is a breaking change as well as the release in WikibaseDataModelSerialization and will break Wikibase.git as well. Should be done in a 2.0

**Comment by addshore** [comment]
Yep, -2 also for the tagging.
If there were no typehints this would not be breaking ;)

File: README.md [hunk 1]
```diff
@@ -106,6 +106,10 @@ Wikibase Internal Serialization has been written by [Jeroen De Dauw]
 
 ## Release notes
 
+## 1.6.0 (2015-08-26)
```
Comment (thiemowmde) [inline README.md]: 3 `###` please.

**Comment by thiemowmde** [comment]
Ouch, this means https://github.com/wmde/WikibaseInternalSerialization/compare/1.4.0...1.5.0 was also a breaking change! Same for https://github.com/wmde/WikibaseDataModelSerialization/compare/1.8.0...1.9.0!

**Comment by addshore** [comment]
yep, I don't think we considered the typehints breaking everything, need to be careful when changing namespaces apparently.
It is a shame PHP doesn't have something like Use class X or use class Y... ;)

**Comment by JeroenDeDauw** [comment]
> Ouch, this means 1.4.0...1.5.0 was also a breaking change!

Since the DM version there was changed, it was not as big of a deal as here

**Pull request closed without merging**